docs/development_guide.md

```diff
@@ -167,9 +167,24 @@ TODO:
 
 ## Helm chart
 
-The Open Service Mesh control plane chart is located in the
+The Open Service Mesh control plane charts are located in the
 [`charts/osm`](../charts/osm) folder.
```

Comment:
Links to files can be absolute based on the root of the repo, so then this link won't break if we move this document:
```suggestion
[`charts/osm`](/charts/osm) folder.
```